contracts/pool/PoolQuotaKeeperV3.sol

```diff
@@ -375,7 +374,7 @@ contract PoolQuotaKeeperV3 is IPoolQuotaKeeperV3, ACLNonReentrantTrait, Contract
 
         // The rate will be set during a general epoch update in the gauge
         quotaTokensSet.add(token); // U:[PQK-5]
-        totalQuotaParams[token].cumulativeIndexLU = uint192(RAY); // U:[PQK-5]
+        totalQuotaParams[token].cumulativeIndexLU = 1; // U:[PQK-5]
```

Comment:
It is an artifact from times when quota index was multiplicative. With additive index, any non-zero number would work (zero indicates that token is not quoted)